scripts/processing/library/config.py

```diff
@@ -19,7 +19,7 @@
     study_path = '/tsi/doctorants/data_gramfort/dgw_faces'
     N_JOBS = 8
 elif user == 'mjas':
-    study_path = '/tsi/doctorants/data_gramfort/dgw_faces'
+    study_path = '/tsi/doctorants/data_gramfort/dgw_faces_reproduce'
```

Comment:
actually this is not required but I can't change it since I am running my scripts on this directory right now